sos/policies/__init__.py

```diff
@@ -726,8 +726,8 @@ def get_local_name(self):
         """Returns the name usd in the pre_work step"""
         return self.host_name()
 
-    def sanitize_case_id(self, case_id):
-        return re.sub(r"[^-a-z,A-Z.0-9]", "", case_id)
+    def sanitize_for_filename(self, name):
```

Comment:
`for_filename` since the method is used to sanitize parts of the tarball filename.

I agree we should sanitize whole filename rather than some of its parts - I will update the PR accordingly and rename the method to `sanitize_filename`, ok?

Reply:
Sure - just looks funny, in that it's actually sanitizing just the label component. I'd agree full-name filtering and `sanitize_filename` looks good.

Looking at the tree as of today, we have:

```python
    def sanitize_report_name(self, report_name):
        return re.sub(r"[^-a-zA-Z.0-9]", "", report_name)

    def sanitize_case_id(self, case_id):
        return re.sub(r"[^-a-z,A-Z.0-9]", "", case_id)
```

Which looks ... weird.

Two almost identical functions checking different parts of the string, and with very slightly different character sets.

Maybe it's time to rationalise this a bit and bring some consistency to it :-)